source/_lovelace/picture-entity.markdown

```diff
@@ -31,7 +31,7 @@ camera_view:
   type: string
 image:
   required: false
-  description: URL of an image.
+  description: URL of an image.<br/>To use a locally hosted image, see [Hosting](/integrations/http#hosting-files).
```

Comment:
We should not use HTML here.

Reply:
```suggestion
  description: URL of an image. To use a locally hosted image, see [Hosting](/integrations/http#hosting-files).
```

Reply:
Oh I see! I copied it from here and thought it was correct:
https://www.home-assistant.io/lovelace/picture-elements/